src/gui_common/AddWindowReorderingSupportToSiblings.cs

```diff
@@ -81,6 +81,11 @@ public class AddWindowReorderingSupportToSiblings : Control
     /// </remarks>
     private readonly HashSet<Node> connectedSiblings = new();
 
+    /// <summary>
+    ///   Used to save windows that ore opened at once to preserve their order.
+    /// </summary>
+    private readonly List<CustomDialog> openedWindows = new();
```

Comment:
```suggestion
    private readonly List<CustomDialog> justOpenedWindows = new();
```
I think renaming this like this will make it clear what this is about.